core-plugins/docs/File-batchsource.md

```diff
@@ -22,6 +22,12 @@ If the format is 'text', the schema must contain a field named 'body' of type 's
 
 **Delimiter:** Delimiter to use when the format is 'delimited'. This will be ignored for other formats.
 
+**Enable Quoted Values** Whether to treat content between quotes as a value. This value will only be used if the format
+is 'csv', 'tsv' or 'delimited'. For example, if this is set to true, a record looks like '1, "a, b, c"' and the delimeter
```

Comment:
For example, if this is set to true, a line that looks like '1, "a, b, c"' will output two fields. The first field will have '1' as its value and the second will have 'a, b, c' as its value. The newline delimiter...